bitnami/schema-registry/templates/_helpers.tpl

```diff
@@ -123,28 +123,28 @@ Compile all warnings into a single message, and call fail.
 
 {{/* Validate values of Schema Registry - SASL authentication */}}
 {{- define "schema-registry.validateValues.authentication.sasl" -}}
-{{- $kafkaProtocol := include "schema-registry.listenerType" ( dict "protocol" (ternary .Values.kafka.auth.clientProtocol .Values.externalKafka.auth.protocol .Values.kafka.enabled) ) -}}
+{{- $kafkaProtocol := upper (ternary .Values.kafka.listeners.client.protocol .Values.externalKafka.listener.protocol .Values.kafka.enabled) -}}
 {{- if .Values.kafka.enabled }}
-{{- if and (contains "SASL" $kafkaProtocol) (or (not .Values.kafka.auth.sasl.jaas.clientUsers) (not .Values.kafka.auth.sasl.jaas.clientPasswords)) }}
-schema-registry: kafka.auth.jaas.clientUsers kafka.auth.jaas.clientPasswords
+{{- if and (contains "SASL" $kafkaProtocol) (or (not .Values.kafka.sasl.client.users) (not .Values.kafka.sasl.client.passwords)) }}
+schema-registry: kafka.sasl.client.users kafka.sasl.client.passwords
     It's mandatory to set the SASL credentials when enabling SASL authentication with Kafka brokers.
     You can specify these credentials setting the parameters below:
-      - kafka.auth.jaas.clientUsers
-      - kafka.auth.jaas.clientPasswords
+      - kafka.sasl.client.users
+      - kafka.sasl.client.passwords
```

Comment:
Should we add these 2 params to the `values.yaml`? Although their equivalents where also missing, it seems strange to me needing to set values for parameters not present in the `values.yaml` file. wdyt?

Reply:
Added this section in the `values.yaml`:
```
  ## @param kafka.sasl.client.users Comma-separated list of usernames for Kafka client listener when SASL is enabled
  ## @param kafka.sasl.client.passwords Comma-separated list of passwords for client listener when SASL is enabled, must match the number of client.users
  ##
  client:
    users:
      - user
    passwords: ""
```